pkg/manifest/build.go

```diff
@@ -40,8 +40,13 @@ func buildResources(fs afero.Fs, o *BuildParameters, m *config.Manifest) (res.Re
 	if err != nil {
 		return nil, err
 	}
-
 	resources = res.Merge(envs, resources)
+	elFiles, err := buildEventlistener(m)
```

Comment:
How about renaming this to clarify that it builds a lot of them?

```suggestion
	elFiles, err := buildEventlistenerResources(m)
```